lib/index.js

```diff
@@ -22,6 +22,9 @@ function register (server, options, next) {
         if (err) {
           return cb(err, null);
         }
+        if (rows.length === 0) { // i.e. no orgs -OR- filter --> wrong tagId or no orgs data
+          return cb(err, null);
```

Comment:
Hi @JMurphyWeb!
The reason why I have the catch `rows.length === 0` is for the case where someone might input an incorrect tagId. If no tag is found at the specified tagId, then `rows` returns an empty array. This then gets passed into `formatter.orgsGetByTag`, which breaks ( #33 ). We need to catch this here, (or we could move it into the formatting function), in order to return the data from the query, in the format that we expect it to be in.